### ISSUE TITLE
git config --global user.{name,email}

### ISSUE THREAD

**Issue by rkdarst** [coderefinery/git-intro#78]
Setting the username seemed to be missing.  This gives big warnings when you first start doing things.  Mention in basics, perhaps - you don't have to go into depth.  It could be introduced either before you do the first commit, or as a reaction when the error message comes up.  git provides good guidance on what to do.

**Comment by rkdarst** [comment]
Could be combined with #74 to make an initial "setup" section - things people should just copy and paste before the lesson starts or knowing what they mean.

**Comment by bast** [comment]
I agree that it should be moved to the lesson and we do it together and explain where the info goes. In every workshop there were 2 participants at least who ran into this.
